### ISSUE TITLE
Missing support for Ubuntu-20.04

### ISSUE THREAD

**Issue by juergenpf** [microsoft/linux-vm-tools#114]
Support for a LTS version should have some priority.

**Comment by ophirharran** [comment]
@juergenpf  - see #106 the fix seems to be in the oven (I tried the proposed fix in this PR and it worked)

**Comment by juergenpf** [comment]
When will this be merged und becoming available in the repo?

**Comment by Hades32** [comment]
You can make it work like this: https://github.com/neutrinolabs/xrdp/issues/1403#issuecomment-559473493

**Comment by trailstrider** [comment]
After some playing around, I got this working, and found that using X2Go over SSH provides a much more robust solution that is also faster to use w/ ssh keys.   Would be awesome if that were built in as an option instead of XRDP.

**Comment by mprachar** [comment]
I have this working, but I am noticing that if I try to do anything that needs sudo while in an enhanced session you get unpredictable results, and/or outright failures. Took me forever to sort out what was happening - basically it is acting like the enhanced session is "outside" the secure session or something,. After turning the enhanced session off all the sudo stuff works fine.

**Comment by duttonw** [comment]
Using the Ubuntu 20.04 image that windows supplies, xrdp did not boot on fresh install. 
to fix this i did the following
```
sudo apt-get update
sudo apt-get upgrade
sudo apt-get purge xrdp
sudo apt-get install xrdp
sudo reboot
```
then rdp worked.  
Oddly (maybe with my corp settings) the enhanced session mode on connect did not work, but using windows remote desktop to the ip address that ubuntu started on allows full screen (multiple monitor ie. 3) as well as clipboard.

**Comment by agashlin** [comment]
@Hades32 's link fixed it for me after an upgrade to 20.04.1. After purging xrdp, I ran install.sh. The xrdp service still failed to start, but worked once I edited these entries in /etc/xrdp/xrdp.ini :

```
port=vsock://-1:3389
use_vsock=false
```

**Comment by cwilhit** [comment]
This repo is being archived, so closing this issue.
